packages/admin/admin/src/form/FieldContainer.tsx

```diff
@@ -152,21 +182,23 @@ const InputContainer = createComponentSlot("div")<FieldContainerClassKey, OwnerS
     componentName: "FormFieldContainer",
     slotName: "inputContainer",
 })(
-    ({ ownerState }) => css`
+    ({ theme, ownerState }) => css`
```

Comment:
```suggestion
    ({ ownerState }) => css`
```

Theme isn't needed here